Atomic/backends/_docker.py

```diff
@@ -362,7 +412,25 @@ def get_dangling_images(self):
         return self._get_images(get_all=True, quiet=True, filters={"dangling": True})
 
     def install(self, image, name, **kwargs):
```

Comment:
Do you think instead of passing this, we should consider call this method rpm_install?